src/pages/groupComparison/comparisonGroupManager/GroupCheckbox.tsx

```diff
@@ -24,6 +24,8 @@ export interface IGroupCheckboxProps {
     markedForDeletion:boolean;
     restore:(group:StudyViewComparisonGroup)=>void;
     rename:(newName:string, currentGroup:StudyViewComparisonGroup)=>void;
+    onDelete:()=>void;
```

Comment:
in keeping with the form of the rest of the props, lets make it `delete:(group:StudyViewComparisonGroup)=>void`

Reply:
doing this makes it easier to make a single function which we pass into `<GroupCheckbox>` elements. This is a React optimization. React only rerenders a component if its props have changed. Anonymous functions (`()=>{}`) are created each time they're used, so they thus change every time, so that means the components props will change every time even if they don't really. Let me know if you have any more questions about this.